wundergraph/README.md

```diff
@@ -0,0 +1,4 @@
+## Environment variables
+
+- `ECOSYSTEM_URL` (optional, default to `http://localhost:4000/graphql`): Link to the ecosystem graphql endpoint.
+- `SWITCHBOARD_URL` (optional, default to `http://localhost:3001/graphql`): Link to the switchboard graphql endpoint.
```

Comment:
Both `SWITCHBOARD_URL` and `ECOSYSTEM_URL` are quite opaque names (I would expect it's URL to a service, not to the graphql endpoint)

Also, note about CORS would be good, since it's a known problem

```suggestion
- `ECOSYSTEM_GQL_ENDPOINT` (required): Link to the graphql endpoint that needs to be migrated (for example `https://countries.trevorblades.com/graphql`)
   - Please note that the underlying service need to set appropriate CORS settings
- `SWITCHBOARD_GQL_ENDPOINT` (optional, default `http://localhost:3001/graphql`): Link to the switchboard graphql endpoint
```